core/templates/pages/exploration-player-page/services/player-position.service.ts

```diff
@@ -41,8 +41,9 @@ export class PlayerPositionService {
     new EventEmitter<HelpCardEventResponse>();
   private _newCardOpenedEventEmitter = new EventEmitter<StateCard>();
 
-  displayedCardIndex = null;
-  onChangeCallback = null;
+  // The following properties are set to null initially.
+  displayedCardIndex: number | null = null;
+  onChangeCallback: Function | null = null;
```

Comment:
Can this be refractored to not use null at all ? We shouldn't use `| null` just because they're initiailized with null